yt_dlp/extractor/facebook.py

```diff
@@ -47,7 +47,7 @@ class FacebookIE(InfoExtractor):
                                 video/embed|
                                 story\.php|
                                 watch(?:/live)?/?
-                            )\?(?:.*?)(?:v|video_id|story_fbid)=|
+                            )\?(?:.*?)(?:v|video_id|story_fbid.*?)=|
```

Comment:
So is this what is being proposed?
```suggestion
                            )\?(?:.*?)(?:v|video_id|id)=|
```
Actually, we could be stricter:
```suggestion
                            )\?(?:.*?&)(?:v|video_id|id)=|
```